modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

```diff
@@ -255,16 +264,18 @@ The Mule runtime engine translates the LATEST dependency keyword to the current
 To obtain the most up-to-date `pom.xml` file information, access 
 the connector in https://www.mulesoft.com/exchange/[Anypoint Exchange] and click *Dependency Snippets*.
 
-== Example: Get, Delete, Query, and Execute
+== Example: Create, Get, Query, Update, and Execute
 
-This example demonstrates the use of SAP S/4 HANA connector to get and delete an entity, query for a value, and execute a function.
+This example demonstrates the use of SAP S/4 HANA connector to create and get an entity, query for a value, update entity and execute a function.
 
 To build and run this demo project, you need:
 
 * Anypoint Studio with at least the Mule 4.1.x Runtime.
 * SAP S/4 HANA Cloud Connector v1.0.0 or later.
 * Credentials for sending ODATA API requests.
 
+NOTE: When using the *Update entity* operation, the entity key _must_ be specified as part of the payload.
+
 === Test the Flow
```

Comment:
Do we need this title given the introduction "The following example..."?
There is a possible disconnect between demonstrating the use of... vs. testing the flow.